lisa/tools/sysbench.py

```diff
@@ -0,0 +1,215 @@
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT license.
+
+from typing import List, cast
+
+from lisa.executable import Tool
+from lisa.operating_system import Posix
+from lisa.util import LisaException
+from lisa.util.process import ExecutableResult
+
+
+class Sysbench(Tool):
+    @property
+    def command(self) -> str:
+        return "sysbench"
+
+    @property
+    def can_install(self) -> bool:
+        return True
+
+    def install(self) -> bool:
+        posix_os: Posix = cast(Posix, self.node.os)
+        posix_os.install_packages("sysbench")
+
+        return self._check_exists()
+
+    def __run_sysbench_test(
+        self,
+        test_type: str,
+        threads: int,
+        events: int,
+        verbosity: int,
+        time_limit: int,
+        percentile: int,
+        debug: str,
+        histogram: str,
+        validation: str,
+        extra_args: str = "",
+    ) -> ExecutableResult:
+        args = (
+            f"{test_type} run --threads={threads} --events={events}"
+            f" --validate={validation} --percentile={percentile}"
+            f" --verbosity={verbosity} --histogram={histogram}"
+            f" --time={time_limit} --debug={debug} {extra_args}"
+        )
+
+        result: ExecutableResult = self.run(args)
+        return result
+
+    def run_cpu_perf(
+        self,
+        threads: int = 1,
+        events: int = 0,
+        cpu_max_prime: int = 10000,
+        verbosity: int = 5,
+        time_limit: int = 10,
+        percentile: int = 95,
+        debug: str = "on",
```

Comment:
It doesn't need to expose all arguments. For example, the debug is always "on", so it should be in the final command arguments only.

Reply:
Default debug would always be on. I wanted to expose all the command line args for sysbench as function parameters so any new requirement where we need to run with say debug=off can be accommodated.

Reply:
LISA is to simplify usage of commands. Only exposing an argument, when it's used in some place.

Reply:
Ok then i will remove some common args with default value as suggested